website/client/src/components/inventory/equipment/index.vue

```diff
@@ -442,9 +430,9 @@ export default {
     ) === CONSTANTS.equipmentDrawerTabValues.COSTUME_TAB;
   },
   methods: {
-    selectDrawerTab (tabName) {
+    selectDrawerTab (index) {
```

Comment:
why this change?

Reply:
i didn't want to change how drawerHeaderTabs works